platform/linux-generic/Makefile.am

```diff
@@ -4,6 +4,8 @@
 include $(top_srcdir)/platform/Makefile.inc
 lib_LTLIBRARIES += $(LIB)/libodp-linux.la
 
+SUBDIRS = scripts
```

Comment:
I'd prefer storing the files under `platform/linux-generic/test`.

Reply:
Ack.